crates/bevy_math/src/primitives/polygon.rs

```diff
@@ -136,6 +153,10 @@ impl Ord for Segment {
 }
 
 /// Holds information about which segment is above and which is below a given [`Segment`]
+#[cfg_attr(
```

Comment:
I thought that we normally use conditional compilation instead of `expect`s.

Reply:
I decided to use `expect` here to communicate that the item itself _is_ `no_std` compatible, it's just not _needed_. But I'm happy to switch to just conditional compilation instead, it should give the same end-results

Reply:
I like your explanation actually